pkg/datapath/linux/routing/routing_test.go

```diff
@@ -38,6 +38,7 @@ type LinuxRoutingSuite struct{}
 var _ = Suite(&LinuxRoutingSuite{})
 
 func (e *LinuxRoutingSuite) TestConfigure(c *C) {
+	c.Skip("Not working")
```

Comment:
Maybe reference issue #11547 in the skip message, so we know why they're not working and sure to remove the skip once the issue is solved.